### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ioszhuk** [avocode/avocode#654]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.2
Source UID: 113287
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by vojtatranta** [comment]
Thank you for your report.
This is known issue and it is duplicated. More info to be found here: #607

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
